packages/kbn-securitysolution-list-api/src/list_api/index.ts

```diff
@@ -70,6 +70,7 @@ const findLists = async ({
   sort_order,
 }: ApiParams & FindListSchemaEncoded): Promise<FoundListSchema> => {
   return http.fetch(`${LIST_URL}/_find`, {
+    version: '2023-10-31',
```

Comment:
FYI, this will conflict with this [PR](https://github.com/elastic/kibana/pull/165658/files#diff-7470327648bd2f2042a9506a83e8081321ba56b2126811934d1b0d56979c1406).

Reply:
Thanks for the heads up 👍 I will remove that change